test/scripts/sigint-env.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// one of these
+process.argv.push('--dprof-no-sigint');
```

Comment:
Oh. When I originally wrote that comment, it was with the understanding that `SIGINT` handling wasn't the default case. So this will cause `dprof` to not dump anything, in which case the file from the previously executed script that did dump exists.

I think you should just remove these two lines, such it does make a dump and call the file `test/scripts/sigint.js`.